security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,28 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-59"]
+== OpenShift Compliance Operator 0.1.59
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.59:
+
+* link:https://access.redhat.com/errata/RHBA-2022:8538[RHBA-2022:8538 - OpenShift Compliance Operator bug fix update]
+
+
+[id="compliance-operator-0-1-59-new-features-and-enhancements"]
+=== New features and enhancements
+
+The Compliance Operator now supports Payment Card Industry Data Security Standard (PCI-DSS) `ocp4-pci-dss` and `ocp4-pci-dss-node` profiles on `ppc64le` alongside `x86_64` architectures. 
+
+[id="compliance-operator-0-1-59-bug-fixes"]
+=== Bug fixes
+
+* Previously, the Compliance Operator did not support the Payment Card Industry Data Security Standard (PCI DSS) `ocp4-pci-dss` and `ocp4-pci-dss-node` profiles on different architectures such as `ppc64le`. Now, the Compliance Operator supports `ocp4-pci-dss` and `ocp4-pci-dss-node` profiles on `ppc64le` architecture. (link:https://issues.redhat.com/browse/OCPBUGS-3252[*OCPBUGS-3252*])
+
+* Previously, after the recent upgrade to version 0.1.57, the `rerunner` service account (SA) was no longer owned by the cluster service version (CSV), which caused the SA to be removed. Now, the CSV owns the `rerunner` SA in 0.1.57. (link:https://issues.redhat.com/browse/OCPBUGS-3452[*OCPBUGS-3452*])
+
+* In 0.1.57, the default metrics port was upgraded; however, the Compliance Operator did not use the new default metrics port. This resulted in `TargetDown` alerts due to the metrics service not connecting to the Compliance Operator's metrics endpoint. With 0.1.59, the Compliance Operator now uses the new default metrics port. (link:https://issues.redhat.com/browse/OCPBUGS-3097[*OCPBUGS-3097*])
```

Comment:
I think this could be clarified some if we mention the port number. I.e.,

In 0.1.57, the operator started the controller metrics endpoint listening on port 8080. This resulted in `TargetDown` alerts since cluster monitoring expected port 8383.  With 0.1.59, the operator starts the endpoint listening on port 8383 as expected.

Reply:
Updated! :)